src/extensions/scratch3_video_sensing/index.js

```diff
@@ -302,59 +328,54 @@ class Scratch3VideoSensingBlocks {
         return motionState;
     }
 
+    static get SensingAttribute () {
+        return SensingAttribute;
+    }
+
     /**
      * An array of choices of whether a reporter should return the frame's
      * motion amount or direction.
-     * @type {object[]} an array of objects.
-     * @param {string} name - the translatable name to display in the drums menu.
-     * @param {string} fileName - the name of the audio file containing the drum sound.
+     * @type {object[]} an array of objects
+     * @param {string} name - the translatable name to display in sensor
+     *   attribute menu
+     * @param {string} value - the serializable value of the attribute
      */
-    get MOTION_DIRECTION_INFO () {
+    get ATTRIBUTE_INFO () {
         return [
             {
-                name: 'motion'
+                name: 'motion',
+                value: SensingAttribute.MOTION
             },
             {
-                name: 'direction'
+                name: 'direction',
+                value: SensingAttribute.DIRECTION
```

Comment:
👍
This may seem a bit silly as-is but this approach will be necessary when we add translation support to all our blocks and menus.